assets/js/eventCardFetcher.js

```diff
@@ -2,7 +2,7 @@ $(function () {
     var api = new GhostContentAPI({
         url: "https://beta.advisory.sg",
         key: "336313a0318904e945f0938d06",
-        version: "v3",
+        version: "v4",
```

Comment:
Were there any API changes from v3 to v4 that might affect this?

Reply:
There doesn't seem to be any related to the content API: https://ghost.org/docs/changes/#ghost-40.